Classes/Driver/CloudinaryFastDriver.php

```diff
@@ -1119,7 +1119,8 @@ public function mergeConfigurationCapabilities($capabilities)
      */
     public function sanitizeFileName($fileName, $charset = '')
     {
-        $fileName = $this->getCharsetConversion()->specCharsToASCII('utf-8', $fileName);
+        $charsetConversion = GeneralUtility::makeInstance(CharsetConverter::class);
```

Comment:
What is the class property `$charsetConversion` for? Is this still needed?

```
    /**
     * @var \TYPO3\CMS\Core\Charset\CharsetConverter
     */
    protected $charsetConversion = null;
```

Reply:
Does this have a negative impact on performance if the CharsetConverter has to be instantiated every time a filename is sanitized? What about using the class property `$charsetConversion` to only do this a single time?

And what about Dependency Injection? Is there a reason it's not used?

Reply:
done, it is customized